src/CommonDBChild.php

```diff
@@ -41,6 +41,8 @@ abstract class CommonDBChild extends CommonDBConnexity
 {
    // Mapping between DB fields
    // * definition
+
+    /** @var class-string<\CommonDBTM> $itemtype */
```

Comment:
This can also be something like `'itemtype'` or `'itemtype_xyz'`. It seems to be the cause of many new PHPStan issues.